### PR TITLE
[terra-status-view] Remove focus from glyphs in IE10

### LINKED ISSUE #3149: [terra-status-view] Status view glyphs are focusable in IE10
# Bug Report

## Description

Status view glyphs are focusable in IE10.

## Expected Behavior

Glyphs should not be focusable

## Possible Solution

Add focusable='false' to the svg element. We should evaluate other components to see if they have a similar problem

### PR DESCRIPTION
**Pull request by saket2403**
### Summary
Removed focus from glyphs in IE10.
Closes #3149 

### Deployment Link

https://terra-core-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Review by ShettyAkarsh — APPROVED** [review]

**Review by avinashg1994 — APPROVED** [review]

File: packages/terra-status-view/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+* Changed
+  * Removed focus from glyphs in IE10 .
+
```
Comment (avinashg1994) [inline packages/terra-status-view/CHANGELOG.md:5]: Should be moved under `Unreleased` section

File: packages/terra-status-view/src/StatusView.jsx [hunk 2]
```diff
@@ -99,7 +99,7 @@ const StatusView = ({
   } else if (variant && !isGlyphHidden) {
     glyphSection = (
       <div className={cx('glyph')}>
-        <svg className={cx(variant)} />
+        <svg focusable={false} className={cx(variant)} />
```
Comment (yuderekyu) [inline packages/terra-status-view/src/StatusView.jsx:102]: lets also recommend this pattern in our documentation for consumers using the custom glyph prop

**Review by jeremyfuksa — APPROVED** [review]

File: packages/terra-status-view/src/StatusView.jsx (hunk 2 above)
Comment (saket2403) [reply packages/terra-status-view/src/StatusView.jsx:103]: Done 322f801

File: packages/terra-status-view/CHANGELOG.md (hunk 1 above)
Comment (saket2403) [reply packages/terra-status-view/CHANGELOG.md]: Done 322f801

**Review by lokesh-0813 — APPROVED** [review]

**Review by emilyrohrbough — APPROVED** [review]

File: packages/terra-status-view/src/StatusView.jsx [hunk 3]
```diff
@@ -99,7 +100,7 @@ const StatusView = ({
...
```
Comment (StephenEsser) [inline packages/terra-status-view/src/StatusView.jsx:103]: Does this need to be an SVG tag? Does switching to a div break/fix expected functionality? On initial glance this is just an empty container that is used to apply a CSS background image.

Comment (saket2403) [reply packages/terra-status-view/src/StatusView.jsx:103]: On replacing svg with div, no image shows up.

**Review by supreethmr — APPROVED** [review]

File: packages/terra-status-view/src/StatusView.jsx (hunk 3 above)
Comment (StephenEsser) [reply packages/terra-status-view/src/StatusView.jsx:103]: Do you know why no image shows up? We use divs/spans for background SVG images throughout the component library. The root issue for this bug is using the empty SVG container, if we need the SVG container then this fix works, but if we don't need the SVG container a more appropriate fix is likely swapping out the SVG tag for a div or span.

Comment (yuderekyu) [reply packages/terra-status-view/src/StatusView.jsx:103]: @saket2403 when you replaced the svg with div, did you also update the style selector? those styles would affect how the markup is rendered: https://github.com/cerner/terra-core/blob/main/packages/terra-status-view/src/StatusView.module.scss#L49

Comment (saket2403) [reply packages/terra-status-view/src/StatusView.jsx]: Updating the css selectors did work, replaced svg with div fe0e024

File: packages/terra-status-view/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
```
Comment (StephenEsser) [inline packages/terra-status-view/CHANGELOG.md:5]: This should be under a Fixed section if the issue was logged as a bug

File: packages/terra-status-view/src/StatusView.jsx [hunk 5]
```diff
@@ -28,6 +28,7 @@ const propTypes = {
 
   /**
    * Display a custom glyph. Overrides a variant's default glyph.
+   * Set `focusable=false` for svg element used as `customGlyph`.
```
Comment (StephenEsser) [inline packages/terra-status-view/src/StatusView.jsx:31]: This comment might not be necessary anymore, but it's not harmful to leave it if users are using a a custom SVG

File: packages/terra-status-view/src/StatusView.jsx [hunk 6]
```diff
@@ -99,7 +100,7 @@ const StatusView = ({
   } else if (variant && !isGlyphHidden) {
     glyphSection = (
       <div className={cx('glyph')}>
-        <svg className={cx(variant)} />
+        <div className={cx(variant, 'bg-common')} />
```
Comment (StephenEsser) [inline packages/terra-status-view/src/StatusView.jsx:103]: We should use a more descriptive class name here, glyph-icon, glyph-container, icon or something more descriptive. bg-common feels a little too generic, almost like a boostrap/global css value

Comment (saket2403) [reply packages/terra-status-view/src/StatusView.jsx]: Changed b469734

File: packages/terra-status-view/CHANGELOG.md (hunk 4 above)
Comment (saket2403) [reply packages/terra-status-view/CHANGELOG.md]: Changed b469734

**Review by pranav300 — APPROVED** [review]

**Review by StephenEsser — APPROVED** [review]

File: packages/terra-status-view/src/StatusView.module.scss [hunk 7]
```diff
@@ -46,7 +46,8 @@
   .glyph {
     padding-bottom: 0.714rem;
 
-    > svg {
+    > svg,
```
Comment (yuderekyu) [inline packages/terra-status-view/src/StatusView.module.scss:49]: this is out of scope and probably a breaking change, but i wonder if we can eventually remove the `svg` selector. currently, it only applies if a consumer passes an svg via the `customGlyph` prop. its probably better if we let consumers handle their own svg styles. 

tho, i think our plan is to eventually remove custom functionality from the status-view in some matter. https://github.com/cerner/terra-core/issues/3012

Comment (emilyrohrbough) [reply packages/terra-status-view/src/StatusView.module.scss:49]: I think you are right that we can probably remove this when we remove the support for custom svgs. It's prob fine for now to leave as is

**Review by yuderekyu — APPROVED** [review]

**Pull request merged**